src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.data.mongodb.host=hk-mongodb
```

Comment:
Get an error because of this when I build

This should not be `hk-mongodb` based on your implementation. It should be `search-mongodb`